resource/ui/ts/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.4.1",
+    "compilerOptions": {
+        "target": "es5",
+        "module": "commonjs",
+        "declaration": false,
+        "noImplicitAny": false,
+        "noEmitOnError": true,
+        "removeComments": true,
+        "noLib": false,
+        "out": "./ui/js/app.js",
+        "sourceMap": false
+    },
+    "files": [
+        "./header.ts",
+        "./app.ts",
+        "./controllers/monitor.ts",
+        "./controllers/rest_explorer.ts"
+    ]
+}
```

Comment:
@BramGruneir there are some configuration options for dealing with whitespace issues in ST. In user settings:
```json
{
	"ensure_newline_at_eof_on_save": true,
        "trim_trailing_white_space_on_save": true
}
```